src/App.test.jsx

```diff
@@ -0,0 +1,15 @@
+import { render } from '@testing-library/react';
+
+import App from './App';
+
+describe('App', () => {
+  it('지역명(버튼)들을 보여준다.', () => {
+    const { container } = render(
+      <App />,
+    );
```

Comment:
```suggestion
    const { container } = render((
      <App />
    ));
```

괄호를 두번감싸주면 콤마를 없앨 수 있어요!

Reply:
넵 감사합니다!!